Lab/01-getting-started.md

```diff
@@ -0,0 +1,18 @@
+# Quickstart
+
+The steps below will get you up and running to deploy your container app to Azure
+
+## Create your own repo from this template by clicking the "Use this Template" button above.
+
+
+![The green "Use this Template" button in the top right section of the repo page on GitHub](../assets/use-this-template.png).
+
+You will be prompted to 
```

Comment:
This is still WIP?

Reply:
Yes... I'm actually going to remove those for this PR